anime-relations.txt

```diff
@@ -35,7 +35,7 @@
 - version: 1.3.0
 
 # Update this date when you add, remove or modify a rule.
-- last_modified: 2019-04-28
+- last_modified: 2019-07-10
```

Comment:
ISO dates are YYYY-MM-DD, I assume you meant 2019-10-07.

Reply:
oops, my bad